charts/opentelemetry-demo/examples/default/rendered/component.yaml

```diff
@@ -1812,10 +1764,6 @@ spec:
         - name: shippingservice
           image: 'ghcr.io/open-telemetry/demo:1.3.0-shippingservice'
           imagePullPolicy: IfNotPresent
-          ports:
-          
-          - containerPort: 8080
```

Comment:
Is the removal of these ports expected?

Reply:
no... I need to look into why this happened

Reply:
this is fixed now